services/app-web/src/components/HealthPlanPackageTable/HealthPlanPackageTable.tsx

```diff
@@ -331,6 +330,12 @@ export const HealthPlanPackageTable = ({
                                 </tr>
                             ))}
                         </tbody>
+                        <caption className={caption ?? styles.visuallyHidden}>
+                            {caption || 'Table'}
+                            <span
```

Comment:
This was moved from above the `<thead>` to the bottom of the `<Table>`. This was done because when visually hiding the `<caption>` there is a thick border below the column headers. Moving `<caption>` to the bottom removed the thick border and when not visually hiding, the caption still displays above the column headers.

Reply:
Also when tabbing into the table, in VO, it will announce the caption twice. Not sure if this was standard, because I would not find anything about this. Navigating into this table using `control + option + directional arrows` does not announce the caption twice.